lib/rules/jsx-filename-extension.js

```diff
@@ -97,6 +103,9 @@ module.exports = {
         }
 
         if (isAllowedExtension && allow === 'as-needed') {
+          if (ignoreFilesWithoutCode && node.body.length === 0) {
```

Comment:
does `node.body` include EmptyStatements, comment nodes, etc?

Reply:
No, it doesn't